scripts/api-extractor.ts

```diff
@@ -0,0 +1,320 @@
+          message.text.includes(
+            'You have changed the public API signature for this project.',
+          )
+        ) {
+          shouldLogInstructions = true;
+          const match = message.text.match(
+            /Please copy the file "(.*)" to "api-report\.md"/,
+          );
+          if (match) {
+            conflictingFile = match[1];
+          }
+        }
+      },
+      compilerState,
+    });
+
+    if (!extractorResult.succeeded) {
+      if (shouldLogInstructions) {
+        console.log('');
+        console.log(
+          '*************************************************************************************',
+        );
+        console.log(
+          '* You have uncommitted changes to the public API of a package.                      *',
+        );
+        console.log(
+          '* To solve this, run `yarn build:api-reports` and commit all api-report.md changes. *',
+        );
+        console.log(
+          '*************************************************************************************',
+        );
+        console.log('');
+
+        if (conflictingFile) {
+          console.log('');
+          console.log(
+            `The conflicting file is ${relativePath(
+              tmpDir,
+              conflictingFile,
+            )}, with the following content:`,
+          );
+          console.log('');
+
+          const content = await fs.readFile(conflictingFile, 'utf8');
+          console.log(content);
+          console.log('');
+        }
+      }
+
+      throw new Error(
+        `API Extractor completed with ${extractorResult.errorCount} errors` +
+          ` and ${extractorResult.warningCount} warnings`,
+      );
+    }
+  }
+}
+
+function isComponentMember(member: any) {
+  // React components are annotated with @components, and we want to skip those
+  return Boolean(member.docComment.match(/\n\s*\**\s*@component/m));
+}
+
+async function buildDocs({
+  inputDir,
+  outputDir,
+}: {
+  inputDir: string;
+  outputDir: string;
+}) {
+  const parseFile = async (filename: string): Promise<any> => {
+    console.log(`Reading ${filename}`);
+    return fs.readJson(resolvePath(inputDir, filename));
+  };
+
+  const filenames = await fs.readdir(inputDir);
+  const serializedPackages = await Promise.all(
+    filenames
+      .filter(filename => filename.match(/\.api\.json$/i))
+      .map(parseFile),
+  );
+
+  const newModel = new ApiModel();
+  for (const serialized of serializedPackages) {
+    serialized.members[0].members = serialized.members[0].members.filter(
+      member => !isComponentMember(member),
+    );
+
+    const pkg = ApiPackage.deserialize(
+      serialized,
+      serialized.metadata,
+    ) as ApiPackage;
+    newModel.addMember(pkg);
+  }
+
+  await fs.remove(outputDir);
+  await fs.ensureDir(outputDir);
+
+  const documenter = new MarkdownDocumenter({
+    apiModel: newModel,
+    documenterConfig: {
+      outputTarget: 'markdown',
+      newlineKind: '\n',
```

Comment:
the other one said `'lf'` :)

Reply:
"De ba dålig kod" :p

Nah the difference is that this is after config processing, the other one is before. This is the value of the enum constant that represents the `'lf'` newline kind.